### DISCUSSION TITLE
ComfyUI icon.

### DISCUSSION THREAD

**Discussion by trustless-zz** [comfyanonymous/ComfyUI#1594] [Show and tell]
I've noticed ComfyUI is missing an icon so I quickly made one for myself. I figured I'd share it here in case anyone would want to use it.
[Adobe Illustrator file.](https://github.com/trustless-zz/ComfyUI_icon/raw/main/comfyui_icon.ai)

![comfyui_r](https://github.com/comfyanonymous/ComfyUI/assets/11897485/8aff6eb9-575f-4fc2-afcf-c6e3cde04d45)
![comfyui_r2](https://github.com/comfyanonymous/ComfyUI/assets/11897485/3f10029b-2119-4391-8458-3379e02418a1)

**Comment by Virgile-fr** [comment]
Awesome, made it into a .ico file :
[ComfyUI.ico.zip](https://github.com/comfyanonymous/ComfyUI/files/14473413/ComfyUI.ico.zip)

**Reply by okareyouok** [reply]
Where should this file be placed? thx. Tried Comfyui/web, but  seems not work

**Reply by Spitfire502** [reply]
Nice—you saved me the trouble of having to do this! I use Red Ketchup to make my ICO files. What do you use for yours?

**Comment by Freehall** [comment]
Good one.

**Comment by scottonanski** [comment]
I made a launcher for it if anyone needs one.

Just tweak it according to your needs;

launcher.sh script;
![carbon1](https://github.com/comfyanonymous/ComfyUI/assets/110937649/6a1e5fd0-d65f-473c-ae72-566e2c40efb1)

.desktop launcher;
![carbon2](https://github.com/comfyanonymous/ComfyUI/assets/110937649/2f1b06e3-cdb3-49f7-9bff-d3cc253f0566)

**Comment by razvanphp** [comment]
> Awesome, made it into a .ico file

Should be added to the frontend, I'm always losing the chrome tab 🙂

**Reply by RalphKaladin** [reply]
Great icons mate, thx.

**Comment by Hiwa5** [comment]
Here is an alternative custom Icon (compressed ico file attached) I made some time ago for my shortcut. Based on the windows folder design and ComfyUI node UI design.
<img width="374" height="336" alt="image" src="https://github.com/user-attachments/assets/cee6dc00-df18-4e03-b4df-721aaa3587e7" />
<img width="501" height="416" alt="image" src="https://github.com/user-attachments/assets/a861f128-8baf-4371-901b-e728b12c8deb" />
on the desk

[ComfyUI-Icon_01.zip](https://github.com/user-attachments/files/21907140/ComfyUI-Icon_01.zip)

**Reply by Metairieman55** [reply]
Used the simple 3 spaghetti one above. Works like a charm. Thanks!

**Reply by AlfredSartan572** [reply]
Just found your icon.
Black-and-white for the console and colored spaghetti for the GUI.
Perfect!
Loving it.
Thank you.

**Comment by pflky** [comment]
This is the one I personally made and use
<img width="1024" height="1024" alt="ComfyUI_ICON" src="https://github.com/user-attachments/assets/025543df-c66c-4913-bacf-726b06ec6bad" />

**Reply by RalphKaladin** [reply]
Thanks!
